financial-templates-lib/logger/Logger.js

```diff
@@ -1,14 +1,18 @@
-// The logger has a number of different levels based on the severity of the incident:
-// -> Debugs: self explanatory. Normal status-based logging. These can trigger
-//   every iteration. Unlimited volume. Prints to console.
-// -> Info: when something happens that is notable, but not necessarily actionable.
-//   These should not trigger every iteration. Any on-chain event that executed correctly.
-//   Print to console & trigger a slack message.
-// -> Error: anything that requires human intervention. If the bot is low on funds or a
-//   transaction fails(some txn failures are sporadic and normal, but it may be difficult
-//   to distinguish).These can trigger every iteration, but only if it's because the bot
-//   encounters a persistent issue that requires human intervention to solve.
-//   Print to console, trigger a slack message and place a phone call to DRI.
+// The logger has a four different levels based on the severity of the incident:
+// -> Debug. It can be considered a console log. Used periodically to inform status updates of repetitive state changes
+//    like polling or no events found. Only viewable on GCE logs.
+// -> Info. Used to report informative events, like a  liquidation/dispute/dispute settlement. These events are
+//    noteworthy but don’t require action or acknowledgment from any
+//    team member. Viewable on GCE logs and sends a slack message to appropriate channels.
+// -> Warn. Used to report warning events that might require a response but don't necessarily indicate system failure.
+//    Require Acknowledgment from the person on duty, or escalation occurs until the warning is acknowledged. For
```

Comment:
nit: this doesn't apply to users outside of UMA. Is it easy for them to hook up these logs to different types of outputs, or at least turn off the PagerDuty integration, etc?

No need to worry too much about this now, just curious how it might work. Also, longer term, it might make sense to have some sort of standard config file to tell the logger how to organize the transports.

Reply:
Right. these don't apply outside of UMA. The great thing about Winston is you can pipe the logs to go anywhere you want. The comment here describes where the logs would go in the given setup. If you wanted you could log these to a text file or a database or use the transports we've provided. It's totally up to you. Added to this the logging levels and their outputs are decoupled so anyone can tune this as they see fit.

In terms of the standard config file this is something I'll be working on in the next few days.